packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -1,6 +1,8 @@
 import assets = require('@aws-cdk/assets');
 import { DockerImageAsset, DockerImageAssetProps } from '@aws-cdk/assets-docker';
 import cloudwatch = require('@aws-cdk/aws-cloudwatch');
+import ec2 = require('@aws-cdk/aws-ec2');
+import { ISecurityGroup } from '@aws-cdk/aws-ec2';
```

Comment:
`ISecurityGroup` should be accessed with the `ec2` prefix as well, and then this import will no longer be required.

Reply:
Done